.env.example

```diff
@@ -1,6 +1,6 @@
 # neccessary Algolia envs for searching to work properly
-ALGOLIA_APPLICATION_ID=YOUR_ALGOLIA_APPLICATION_ID
-ALGOLIA_API_KEY=YOUR_ALGOLIA_API_KEY
+ALGOLIA_APPLICATION_ID=YHFN1WRCR5
+ALGOLIA_API_KEY=5981bf15ffc7630353478168f2574228
```

Comment:
??? Is this supposed to be public?!

Reply:
Argh, how did that get in there.

Reply:
Fixing it ended up in the example and not in .env.

Reply:
Removed. It's only inside of thie PR note. I removed it from git history and the repo is not public.

Reply:
It's actually a public key for search anyways, but better not in the repo.